deploy/chart/templates/deployment.yaml

```diff
@@ -23,18 +23,28 @@ spec:
       serviceAccountName: {{ template "local-path-provisioner.serviceAccountName" . }}
       containers:
         - name: {{ .Chart.Name }}
+        {{- if .Values.privateRegistry.registryUrl }}
+          image: "{{ .Values.privateRegistry.registryUrl }}/{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+        {{- else }}
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+        {{- end }}
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           command:
             - local-path-provisioner
             - --debug
             - start
             - --config
             - /etc/config/config.json
+            - --service-account-name
+            - {{ template "local-path-provisioner.serviceAccountName" . }}
```

Comment:
this is not working with current docker image: `rancher/local-path-provisioner:v0.0.17`:
```bash
# k logs local-path-provisioner-7ccbc4dfc9-sft56
Incorrect Usage: flag provided but not defined: -service-account-name
NAME:
   local-path-provisioner start - 
USAGE:
   local-path-provisioner start [command options] [arguments...]
OPTIONS:
   --config value            Required. Provisioner configuration file.
   --provisioner-name value  Required. Specify Provisioner name. (default: "rancher.io/local-path") [$PROVISIONER_NAME]
   --namespace value         Required. The namespace that Provisioner is running in (default: "local-path-storage") [$POD_NAMESPACE]
   --helper-image value      Required. The helper image used for create/delete directories on the host (default: "busybox") [$HELPER_IMAGE]
   --kubeconfig value        Paths to a kubeconfig. Only required when it is out-of-cluster.
   --configmap-name value    Required. Specify configmap name. (default: "local-path-config")
   
time="2020-09-09T12:31:07Z" level=fatal msg="Critical error: flag provided but not defined: -service-account-name" 
```

latest working commit for helm chart is `e4dfa34ae06b57667a854b216b5eba6f62f94cb7`

Reply:
Hi, currently only working on master. v0.0.17 does not support the sa option